### ISSUE TITLE
4 icons missing on the clapper interface

### ISSUE THREAD

**Issue by bluebyt65** [Rafostar/clapper#9]
4 icons missing (Floating mode, full screen and both icons near the volume)
ArchLinux, Gnome 3.38.1 on Wayland
Installed clapper with Flatpak
Intel® HD Graphics 2500 (IVB GT1)

![slurped](https://user-images.githubusercontent.com/18442224/99526757-b7fc4400-2969-11eb-8463-4a3ce8657242.png)

**Comment by Rafostar** [comment]
Ah, so its just the icons. When you originally mentioned this on `reddit` I thought that the whole buttons were missing (cause I have some hide or show them logic in my code). My mistake :smile: 

Anyway, the icons are not part of this application, but taken from the icon theme. What icon theme are you using? If you switch to Adwaita icon theme before starting the player do you get those icons?

The icons that seem to be missing are:
* preferences-desktop-remote-desktop-symbolic
* view-fullscreen-symbolic
* emblem-videos-symbolic
* emblem-music-symbolic

**Comment by bluebyt65** [comment]
Yes if I change to Adwaita icon theme everything is good, You can close this if you want, I understand its not a bug.

**Comment by Rafostar** [comment]
Since default GNOME theme has them, you should file this as a missing icons bug for the icon theme you are using and kindly ask them to add the missing icons I listed above. Good luck!
